DataStructures/BitArray/BitArray.cs

```diff
@@ -0,0 +1,1220 @@
+﻿/**
+ * Author: Christian Bender
```

Comment:
Please remove the author information from this class. We maintain a high-level list of contributors in the README.md file.

Reply:
OK, I will changed the source code and then pull again